lib/src/client.dart

```diff
@@ -495,11 +512,9 @@ class Client extends MatrixApi {
       homeserver = homeserverUrl.stripTrailingSlash();
 
       // Look up well known
-      DiscoveryInformation? wellKnown;
       if (checkWellKnown) {
         try {
-          wellKnown = await getWellknown();
-          homeserver = wellKnown.mHomeserver.baseUrl.stripTrailingSlash();
```

Comment:
This will change the behavior of `client.checkHomeserver(domain)` as this will now no longer connect to the baseUrl in the well-known file. As a client-developer, I just want to pass the input of the homeserver-textfield to `client.checkHomeserver(input)` and let the sdk handle this

Reply:
Ah, right, forgot to re-add it - I wanted to move it to the `getWellKnown` method but apparently forgot about it. Thanks.

Reply:
Fixed it back.